README.md

```diff
@@ -111,7 +111,31 @@ import About from 'pages/About'
 
 ## Config
 
-### TCR (Optional)
+Default app configs can be found on [the default config file](./config-default.yaml)
+
+We recommend against editing this file directly, though.
+
+If you wish to replace any default config, create a file named `config` inside the [`custom` folder](./custom).
+
+Both JSON and YAML formats are supported.
+
+Simply replace any config found on [config-default](./config-default.yaml).
+
+Below we provide details for each config.
+
+### `name`
```

Comment:
Maybe we can use a table like markdown? showing the name, the description, the default value

Reply:
I continued reading, and a table maybe makes it more complicated for the TCR part, cause it's a bit complex. So maybe just ignore my comment if there's no easy way to make a compact description of the parameters

Ultimately, I think we might find ourselves replicating between the README.md and the `config-default.yaml` so it's not bad to consider being brief in the readme, and use comments in the default file to document things if we come to that point.

Reply:
We can split this into a separated readme/wiki/etc for config if this becomes to big.
For me this is still manageable. I've seem much larger readmes out there :)

Reply:
Cool!